app/models/concerns/push_updates_minimal.rb

```diff
@@ -0,0 +1,50 @@
+module PushUpdatesMinimal
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    attr_reader :target_channels
+    attr_reader :target_channels_func
+
+    private
+
+    def push_targets(channels = [], &block)
+      @target_channels = channels
+      if block_given?
+        @target_channels_func = block
+      end
+    end
+  end
+
+  def push_changes
+    PushService.new.send_update_store target_channels, {
+      item: serialize(self),
+      sender: user,
+      operation: read_operation(self)
+    }
+  end
+
+  def target_channels
+    if self.class.target_channels_func
+      return self.class.target_channels_func.call(self)
+    end
+    self.class.target_channels
+  end
+
+  private
+
+  def serialize(record)
+    associations = record.class.reflections.keys.map(&:to_sym)
+    "Api::V1::#{record.class}Serializer".constantize.new(record, { exclude: associations })
```

Comment:
Will suggest using`safe_constantize` instead of `constantize`.

https://apidock.com/rails/String/safe_constantize

Reply:
sure, I just copied that from the other files

Reply:
I've updated it 👍

Reply:
👍